pet-store-react-hooks/src/infrastructure/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -12,7 +12,8 @@ export function ErrorMessage(props: ErrorMessageProps) {
 
   return (
     <div
-      aria-label="system-error-message"
+      role="alert"
+      aria-label="system error message"
```

Comment:
No need to set aria-label here: it inherits it from it's contents.

Reply:
In the tests i get the error message by: 
```
const errorMessage = await screen.findByRole('alert', {
    name: 'system error message',
});
```
If i remove the aria-label, how can i be more specific about the role of alert that is found?
Do i just leave it:
```
const errorMessage = await screen.findByRole('alert');
expect(errorMessage).toHaveTextContent(
  'System error. Please contact the system administrator.'
);
```

Reply:
Yes. You only need to set accessibility labels if there is no other option accessibility-wise. You never add accessibility labels just for testing.